FlashLFQ/Bayesian Protein Quantification/ProteinQuantificationEngineResult.cs

```diff
@@ -8,125 +8,105 @@ namespace FlashLFQ
     public class ProteinQuantificationEngineResult
     {
         public readonly ProteinGroup protein;
-        public readonly string condition1;
-        public readonly string condition2;
+        public readonly string BaseCondition;
```

Comment:
Does this only work for binary? Can it be extended to more conditions? In drug trials on cells one usually has control, vehicle and treatment.

Reply:
right now it's only for binary comparisons. you can call the engine multiple times with different base conditions if you want an all vs. all comparison. at some point it would be cool to do a bayesian ANOVA but that's probably down the line a ways